packaging/README.release.md

```diff
@@ -1,11 +1,10 @@
 # Creating a New Release
```

Comment:
Please change this file to mention that the issue template should be used for doing a new release.